tests/curators/test_curators_general.py

```diff
@@ -235,3 +239,40 @@ def test_schema_ordered_set(df):
     # clean up
     ln.Schema.filter().delete()
     ln.Feature.filter().delete()
+
+
+def test_schema_no_minimal_set_var():
+    """If minimal_set is False, invalid ensembl gene IDs are allowed."""
```

Comment:
Wait, now I'm confused. Why does this test refer to to minimal set and not to maximal set?